### ISSUE TITLE
Continuity of Operations >> Arrangements for Virtual Presence >> Skimmability

### ISSUE THREAD

**Issue by brewerj** [w3c/Guide#68]
General comment on this section: Most people will be skimming this, and it's hard to follow. I suggest adding a topic word or two at the beginning of each paragraph to clarify the organization of this section and make it easier to find relevant information. Given current content, you could consider: "Use of video conferencing services" "Scheduling" "Materials" etc. 

Additionally, add a topical paragraph, either with the sub-head "Web For All" or specifically "Accessibility." I am guessing that there are I18N, privacy, and security considerations that people could be briefly reminded of here, which would favor a "Web for All" topic word. Definitely there are accessibility considerations. We will be developing this further, but for now, please use: "Accessibility: Please ensure that virtual presence meetings are accessible for participants. Check needs in advance with participants. Use WAI resources and/or contact W3C staff for questions as needed."

**Comment by brewerj** [comment]
For this section "Arrangements for Virtual Presence," if you make this section skimmable as suggested above, you could move the last paragraph in the previous section into this section, to make it easier to find. Topic word(s) at the start of paragraph could be "Supplemental asynchronous discussion", e.g. (with a few proposed tweaks):

"Asynchronous supplemental discussion: For all types of sessions, the Chairs should also allow asynchronous feedback and discussion, using GitHub issues, emails, or other forms of input before and after a virtual presence meeting, or between meetings in the case of a multi-day meeting."

(BTW, I experimented with this during a multi-day virtual meeting last week, and it was promising.)

**Comment by brewerj** [comment]
Last structural suggestion for this section: consider boxing/highlighting the contents of this section, instead of the contents of the previous section. This section seems to have more of the practical bits of how to make a meeting work.

**Comment by swickr** [comment]
Addressed in https://github.com/w3c/Guide/commit/274133574560fefc0d7b8b0b940f951b11a77600 except for the 'boxing' comment.  I don't fully grok that suggestion.

**Comment by brewerj** [comment]
Thanks @swickr for structuring the "Arrangements" section. 
The info in the "Arrangements" section may be more practical and useful for readers than the information in the previous section, which is boxed and highlighted. I'm unclear what the purpose of that section is -- perhaps it is to present a taxonomy of meeting types? Somewhat confusing to read as it is unclear whether it covers the case of a normal meeting.
I was suggesting highlighting the material in the "arrangements" section as well, or instead. 
Not high priority, feel free to close without addressing.

**Comment by brewerj** [comment]
The reorganization and repositioning of the "Arrangements" section appears to have increased the skimmability of key practical information. Thx. The top-level advice to allow room to discuss logistics is also useful. 
IMO it could still be made clearer by separating out the general point and the accessibility-specific guidance. Reorganizing and revising what you have: 
    "Virtual presence meetings occur when all or some of the participants are attending remotely. All participants should realize that when people pivot to new environments, new communication and interoperability issues can arise for any participants. Meeting chairs should leave space for the participants to discuss this, and to ensure that accessibility needs of our participants are addressed."
    This first mention of "accessibility" could then link down to a slightly expanded accessibility subsection, e.g.: 
    "**Accessibility**: In all cases W3C wants to make sure that people with accessibility needs can participate. Meeting organizers must consider how best to accomplish that with what is currently available; e.g. is captioning or interpretation needed? Are meeting organizers aware of any potential interoperability issues between the videoconferencing platform and individuals' assistive technologies? Ensure that virtual presence meetings are accessible for all participants, using resources including [Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/) and [How to Meet WCAG 2, A Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/)."

**Comment by swickr** [comment]
Thanks @brewerj; I adopted this suggestion in https://github.com/w3c/Guide/commit/e43bb70ebede0ac5c2a6b32e5f1058b48e277428

**Comment by brewerj** [comment]
LGTM, thx
